packages/analytics-js-plugins/src/utilities/retryQueue/index.ts

```diff
@@ -462,14 +461,12 @@ class Queue extends Emitter {
         }
 
         res.push(
-          new Store(
-            {
-              name,
-              id: parts[1],
-              validKeys: QueueStatuses,
-            },
-            getStorageEngine('localStorage'),
-          ),
+          this.storeManager.setStore({
+            id: parts[1],
+            name,
+            validKeys: QueueStatuses,
+            type: 'localStorage',
```

Comment:
Same as above.

Reply:
on it

Reply:
seems that this hardcodded value is as expected asreclaim only happens from localstrage